services/account_type/master_payer.yaml

```diff
@@ -0,0 +1,121 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+
+Description: Lists the cloudtrails in the current account
+
+
+Parameters:
+  IsMasterPayerAccount:
+    Description: Flag to indicate if this is the master payer account
+    Type: String
+    AllowedValues:
+      - 'true'
+      - 'false'
+  MasterPayerBillingBucketName:
+    Description: The name of the S3 bucket responsible for billing data
+    Type: String
+  ReactorAccountId:
+    Description: The CloudZero reactor AWS account ID
+    Type: String
+    Default: '061190967865'
+  ExternalId:
+    Description: |
+      Unique ExternalId for Customer Organization; for cross-account Role Access and
+      associating this template with a Customer Organization
+    Type: String
+
+
+Conditions:
+  CreateResources: !And [ !Not [ !Equals [ !Ref MasterPayerBillingBucketName, 'null' ] ], !Equals [ !Ref IsMasterPayerAccount, 'true' ] ]
+
+
+Resources:
+
+  CrossAccountRole:
+    Condition: CreateResources
+    Type: 'AWS::IAM::Role'
+    Properties:
+      RoleName: !Sub 'cz-reactor-access-${ReactorAccountId}'
+      Path: '/'
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: czServiceAccount01
+            Effect: Allow
+            Principal:
+              AWS: !Sub 'arn:aws:iam::${AWS::AccountId}:root'
+            Action:
+              - 'sts:AssumeRole'
+            Condition:
+              StringEquals:
+                'sts:ExternalId': !Ref ExternalId
+          - Sid: czVpcFlowLogs01
+            Effect: Allow
+            Principal:
+              Service: 'vpc-flow-logs.amazonaws.com'
+            Action: 'sts:AssumeRole'
+      ManagedPolicyArns:
+        - 'arn:aws:iam::aws:policy/SecurityAudit'
+        - 'arn:aws:iam::aws:policy/AmazonSNSReadOnlyAccess'
+        - 'arn:aws:iam::aws:policy/CloudWatchReadOnlyAccess'
+        - 'arn:aws:iam::aws:policy/CloudWatchEventsReadOnlyAccess'
+        - 'arn:aws:iam::aws:policy/AWSXrayFullAccess'
+        - 'arn:aws:iam::aws:policy/AmazonGuardDutyReadOnlyAccess'
+        - 'arn:aws:iam::aws:policy/AmazonInspectorReadOnlyAccess'
+        - 'arn:aws:iam::aws:policy/AmazonRoute53ReadOnlyAccess'
+
+  CrossAccountRoleAccessPolicyPayer:
+    Condition: CreateResources
+    Type: 'AWS::IAM::Policy'
+    Properties:
+      PolicyName: CrossAccountRoleAccessPolicyPayer
```

Comment:
Please update `PolicyName` to something like `PolicyName: !Sub 'cloudzero-master-payer-policy-${ReactorAccountId}'`